library/connect.c

```diff
@@ -482,8 +523,16 @@ int ziti_accept(nf_connection conn, nf_conn_cb cb, nf_data_cb data_cb) {
     req->channel = conn->channel;
     req->conn = conn;
     req->cb = cb;
+    LIST_INSERT_HEAD(&conn->nf_ctx->connect_requests, req, _next);
+
     ziti_channel_send_for_reply(ch, content_type, headers, 3, (const uint8_t *) &clt_conn_id, sizeof(clt_conn_id),
                                 connect_reply_cb, req);
 
     return ZITI_OK;
 }
+
+int ziti_process_connect_reqs(nf_context nf) {
+    ZITI_LOG(WARN, "TODO");
```

Comment:
maybe leave a link to an issue here instead?

Reply:
this is not used yet. Maybe never will. I'll resolve one way or the other